src/deploy/functions/backend.ts

```
@@ -542,21 +543,16 @@ async function loadExistingBackend(ctx: Context): Promise<Backend> {
   }
   unreachableRegions.gcfV1 = gcfV1Results.unreachable;
 
-  let gcfV2Results;
   try {
-    gcfV2Results = await gcfV2.listAllFunctions(ctx.projectId);
-    for (const apiFunction of gcfV2Results.functions) {
-      const endpoint = gcfV2.endpointFromFunction(apiFunction);
+    const runServices = await run.listServices(ctx.projectId);
+    for (const service of runServices) {
+      const endpoint = run.endpointFromService(service);
       existingBackend.endpoints[endpoint.region] = existingBackend.endpoints[endpoint.region] || {};
       existingBackend.endpoints[endpoint.region][endpoint.id] = endpoint;
     }
-    unreachableRegions.gcfV2 = gcfV2Results.unreachable;
   } catch (err: any) {
-    if (err.status === 404 && err.message?.toLowerCase().includes("method not found")) {
-      // customer has preview enabled without allowlist set
-    } else {
-      throw err;
-    }
+    logger.debug(err.message);
+    unreachableRegions.run = ["unknown"]; // Indicate that Run services could not be listed
```

q: why do we ignore errors? e.g. what if 403 is thrown?

---

Accidentally removed above, added back in. I think previous case only handled 404 - is this ok to keep as is or should I add 403 error handling as well?